kotlin/README.md

```diff
@@ -0,0 +1,111 @@
+# Looker SDK
+
+The Looker SDK for Kotlin provides a convenient way to communicate with the Looker API available on your Looker server. 
+
+**DISCLAIMER**: This is an _alpha_ version of the Looker SDK, using a completely new code generator developed by Looker. Implementations are still subject to major change. If you run into problems with the SDK, feel free to [report an issue](https://github.com/looker-open-source/sdk-codegen/issues), and please indicate which language SDK you're using in the report.
+
+## Getting started
+
+The Looker SDK can be used in a Kotlin application in 3 steps:
+
+* configure
+* install
+* use
+
+### Configure the SDK for your Looker server
+
+**Note**: The `.ini` configuration for the Looker SDK is a sample implementation intended to speed up the initial development of Node applications using the Looker API. See this note on [Securing your SDK Credentials](https://github.com/looker-open-source/sdk-codegen/blob/master/README.md#securing-your-sdk-credentials) for warnings about using `.ini` files that contain your API credentials in a source code repository or production environment.
+
+Create a `looker.ini` file with your server URL and API credentials assigned as shown in this example.
+
+```ini
+[Looker]
+# API version defaults to 3.1. 3.1 and 3.0 are currently supported. 3.1 is highly recommended.
+api_version=3.1
+# Base URL for API. Do not include /api/* in the url
+base_url=https://<your-looker-server>:19999
+# API 3 client id
+client_id=your_API3_client_id
+# API 3 client secret
+client_secret=your_API3_client_secret
+```
+
+**Note**: If the application using the Looker SDK is going to be committed to a version control system, be sure to
+**ignore** the `looker.ini` file so the API credentials aren't unintentionally published.
+
+### Install the Looker SDK for Kotlin
+
+The alpha version of the Looker SDK is not published to a Kotlin Package Manager. It's way too early for that. Currently, the only way to get the source code is by cloning the Looker SDK Codegen repository and use the source code in the `kotlin` folder.
+
+To ensure you have the version of the SDK that matches your Looker version, you can regenerate `methods.kt` and `models.kt` from the root of the repository with the command:
+
+```bash
+yarn sdk kotlin
+```
+
+If this command fails the first time, read the [instructions for setting up `yarn`](https://github.com/looker-open-source/sdk-codegen/blob/master/README.md#using-the-yarnnode-based-generator)
+
+### Use the SDK in your code
+
+When the SDK is installed and the server location and API credentials are configured in your `looker.ini` file, it's ready to be used.
+
+Verify authentication works and that API calls will succeed with code similar to the following:
+
+```kotlin
+val localIni = "/Users/Looker/sdk-codegen/looker.ini"
+val settings = ApiSettingsIniFile(localIni, "Looker")
+
+val client: HttpClient = HttpClient(Apache) {
```

Comment:
This is only necessary when validating against a Looker install with a self-signed cert. Do our customer instances have non-self-signed certs? Regardless, I think it's probably worth noting here that the `TrustSelfSignedStrategy` and `NoopHostnameVerifier` are dangerous and that they likely _should not_ end up in anything that is production level code.

Reply:
Thanks, replaced this code sample